projects/igniteui-angular/migrations/update-13_0_0/index.ts

```diff
@@ -61,5 +63,22 @@ export default (): Rule => (host: Tree, context: SchematicContext) => {
             }
         });
     }
+
+    for (const path of moduleTsFiles) {
+        let content = host.read(path)?.toString();
+        const servicesInFile = [];
+        SERVICES.forEach(service => {
+            if (content.indexOf(service) > -1) {
+                servicesInFile.push(service);
+            }
+        });
+
+        if (servicesInFile.length > 0) {
+            const comment = '// ' + servicesInFile.join(' and ') + ' no longer need to be manually provided and can be safely removed.';
+            content = comment + content;
```

Comment:
Might be missing something here, but if you directly prepend the comment to the content - shouldn't there be a new line somewhere?